opba-protocols/xs2a-protocol-tests/xs2a-bdd-sandbox/src/test/java/de/adorsys/opba/protocol/xs2a/tests/e2e/sandbox/SandboxE2EProtocolTest.java

```diff
@@ -184,6 +188,66 @@ void testTransactionsListWithConsentUsingEmbedded(Approach expectedApproach) {
             );
     }
 
+    @ParameterizedTest
+    @EnumSource(Approach.class)
+    void testSinglePaymentUsingEmbedded(Approach expectedApproach) {
+        given()
+            .enabled_embedded_sandbox_mode()
+            .preferred_sca_approach_selected_for_all_banks_in_opba(expectedApproach)
+            .rest_assured_points_to_opba_server()
+            .user_registered_in_opba_with_credentials(OPBA_LOGIN, OPBA_PASSWORD);
+        when()
+            .fintech_calls_initiate_payment_for_max_musterman()
+            .and()
+            .user_logged_in_into_opba_as_opba_user_with_credentials_using_fintech_supplied_url_pis(OPBA_LOGIN, OPBA_PASSWORD)
+            .and()
+            .user_max_musterman_provided_initial_parameters_to_list_accounts_all_accounts_consent()
```

Comment:
Albeit payment API uses same steps as Consent API, can we add specialized methods like
`user_max_musterman_provided_initial_parameters_to_make_payment`
instead of 
`user_max_musterman_provided_initial_parameters_to_list_accounts_all_accounts_consent`
and 
`user_max_musterman_provided_initial_parameters_to_make_payment` simply calls `user_max_musterman_provided_initial_parameters_to_list_accounts_all_accounts_consent` under the hood

Reply:
fixed